client/src/components/login/login.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+
+export default class Login extends React.Component {
+  render () {
+    return (
+      <div>
+        <div className='col-md-3 col-md-offset-4'>
+          <div className='form-group'>
+            <h3>Sign In:</h3>
+            <form>
+              <label for='inputUsername'>Username</label><br />
+              <input type='username' className='form-control' id='inputUsername' placeholder='Username' />
```

Comment:
For the `id`s here, something like `username` would be just fine, and is a bit more concise than `inputUsername`. @michael-beaver might also have some thoughts on that, so maybe wait before you change that.